src/plugins/intel_gpu/src/kernel_selector/common_types.h

```diff
@@ -492,6 +492,8 @@ enum class ScatterUpdateAxis {
     Y,
     Z,
     W,
+    U,
```

Comment:
Why don't you add unit test case using the scatter update issue which first kernel is skipped  ?

Reply:
I think it is hard to reproduce the case in unit test.
Condition 1) axis = batch or feature
Condition 2) output buffer should be shared with other output node

Reply:
I think condition2 is resolved by adding output memory with dummy data to scatter_update. because If first kernel is not called, scatter update does not return expected result.

Reply:
Added unit test. Thanks!